frappe/integrations/doctype/razorpay_settings/razorpay_settings.py

```diff
@@ -317,6 +319,20 @@ def cancel_subscription(self, subscription_id):
 		except Exception:
 			frappe.log_error(frappe.get_traceback())
 
+	def verify_signature(self, body, signature, key):
```

Comment:
where is this method to be called from?

Reply:
https://github.com/frappe/erpnext/pull/21872/files